components/outer-layout/server.js

```diff
@@ -110,6 +111,21 @@ export class OuterLayout extends ServerComponent {
    * @param {import("@fred").Context} context
    */
   _renderMeta(context) {
+    const ALWAYS_ALLOW_ROBOTS = false; // FIXME
```

Comment:
@LeoMcA How should I pass this here from the environment?

See: https://github.com/search?q=repo%3Amdn%2Fyari+%2FALWAYS_ALLOW_ROBOTS%2F+path%3A%2F%5E%5C.github%5C%2Fworkflows%5C%2F%2F&type=code

Reply:
Make it something like `FRED_ALLOW_ROBOTS` (prefixed with `FRED_`) then add it to the env component: https://github.com/mdn/fred/blob/e2aceed83a39d2716fbe375d9258a2ab38468678/components/env/index.js

Reply:
Gotcha, done in 918f315.